CalibPPS/ESProducers/python/ctppsLHCInfo_cff.py

```diff
@@ -1,7 +1,6 @@
 import FWCore.ParameterSet.Config as cms
```

Comment:
my suggestion would be to remove this file completely and whatever used it could take the fillDescription based auto-generated cfi file

Reply:
addressed in 898bf11e0f06508b452854a518f42f9fbb151882